CHANGELOG.md

```diff
@@ -11,8 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -
 -
 ### Changed
-
--
+ 
+- Impose specific order of columns on return value of TSDataset.to_flatten ([#873](https://github.com/tinkoff-ai/etna/pull/873))
```

Comment:
Sorry, I forgot to mention, we set here number of PR, not the number of issue. It will be 1095 in your case, not 873.

Reply:
Fixed it)